File: client-app/src/app/models/patient.ts
export interface Patient {
    id: string;
    firstName: string;
    lastName: string;
    age: string;
    registeredSince: string;
    email: string;
    status:string;
    role: string;
    username: string;
    diagnosis: string;
}

export interface PatientTable {
    firstName: string;
    lastName: string;
    age: string;
    email: string;
    status: string;
    // registeredSince: string;
    username: string;
    role: string;
    diagnosis: string;
}

File: Domain/Patient.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class Patient
    {
        public Guid Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int Age { get; set; }
        public DateTime RegisteredSince { get; set; }
        public string Email { get; set; }
        public string Status { get; set; }

    }
}